[SolarWinds] Add `Port` parameter (#38517)
demisto/content: Packs/SolarWinds/ReleaseNotes/1_0_18.md

#### Integrations

##### SolarWinds
- Updated the Docker image to: *demisto/python3:3.12.8.1983910*.

> - Added support for **Port** parameter with which to connect to the orion server. examples: 17778, 17774.

Reviewer: ```suggestion
- Added the **Port** parameter which is used to connect to the Orion server. For examples: 17778, 17774.
```
